core/src/main/java/com/google/errorprone/refaster/ExpressionTemplate.java

```diff
@@ -282,7 +282,7 @@ private static int getPrecedence(JCTree leaf, Context context) {
       return (typeCast.expr == leaf) ? TreeInfo.prefixPrec : TreeInfo.noPrec;
     } else if (parent instanceof JCInstanceOf) {
       JCInstanceOf instanceOf = (JCInstanceOf) parent;
-      return TreeInfo.ordPrec + ((instanceOf.clazz == leaf) ? 1 : 0);
+      return TreeInfo.ordPrec + ((instanceOf.getType() == leaf) ? 1 : 0);
```

Comment:
Similar as below, the field has been removed and we should use `getType()` (which is a getter for the field).